corehq/messaging/management/commands/resync_case_type.py

```diff
@@ -0,0 +1,25 @@
+from corehq.form_processor.interfaces.dbaccessors import CaseAccessors
+from corehq.messaging.tasks import sync_case_for_messaging
+from corehq.util.log import with_progress_bar
+from django.core.management.base import BaseCommand, CommandError
+
+
+class Command(BaseCommand):
+    help = "Sync messaging models for cases"
+
+    def add_arguments(self, parser):
+        parser.add_argument('domain')
+        parser.add_argument('case_type')
+        parser.add_argument('--limit', type=int, default=-1)
+
+    def handle(self, domain, case_type, limit, **options):
+        print("Fetching case ids for %s/%s ..." % (domain, case_type))
+        case_ids = CaseAccessors(domain).get_case_ids_in_domain(case_type)
```

Comment:
this is for ccs_record, i've used this approach for child_health in the past (which has more than ccs_record) and it went ok. i haven't had to do it for person yet